pyba/Camera.py

```diff
@@ -142,29 +142,39 @@ def summarize(self):
     def plot_2d(
         self,
         img_id: int,
-        points2d: Optional[np.ndarray] = None,
+        points: Optional[np.ndarray] = None,
         bones: Optional[np.ndarray] = None,
         colors: Optional[List[Tuple]] = None,
     ) -> np.ndarray:
-        img = self.get_image(img_id)
-        points2d = self.points2d[img_id] if points2d is None else points2d
+        """
+        Parameters
+        ----------
+        ...
+
+        points: either points2d to plot directly, or points3d in which case they will
+        be projected for plotting.
 
+        """
+        img = self.get_image(img_id)
+        points = self.points2d[img_id] if points is None else points[[img_id]]
```

Comment:
Do we need the double brackets `[[img_id]]`? If not, change to single

Reply:
Yes I have the same question

Reply:
yes i have the same question